ae_network_connection_exception.gemspec

```diff
@@ -1,21 +1,26 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'ae_network_connection_exception/version'
+# frozen_string_literal: true
+
+require_relative 'lib/ae_network_connection_exception/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "ae_network_connection_exception"
+  spec.name          = 'ae_network_connection_exception'
   spec.version       = AeNetworkConnectionException::VERSION
-  spec.authors       = ["Steven Boyd"]
-  spec.email         = ["engineering@appfolio.com"]
-  spec.summary       = "Provides sane exceptions for network failures"
-  spec.description   = "A simple gem that will catch all manner of exceptions related to establishing a network connection and return a more generic error."
-  spec.homepage      = "http://github.com/appfolio/ae_network_connection_exception"
-  spec.license       = "MIT"
-
-  spec.files         = `git ls-files -z`.split("\x0")
-  spec.require_paths = ["lib"]
+  spec.author        = 'Appfolio'
+  spec.email         = 'dev@appfolio.com'
+  spec.summary       = 'Provides sane exceptions for network failures'
+  spec.description   = 'Catch exceptions related to establishing a network connection and return a generic error.'
+  spec.homepage      = 'http://github.com/appfolio/ae_network_connection_exception'
+  spec.license       = 'MIT'
+  spec.files         = Dir['**/*'].select { |f| f[%r{^(lib/|Gemfile$|Rakefile|README.md|LICENSE.txt|.*gemspec)}] }
```

Comment:
I think the trailing `$` needs to be added for the other filenames as well.
I would also add `test/`.
I would remove `Gemfile`.
I would spell out the gemspec filename or write it as `\w+\.gemspec`.

Reply:
This will be the new list to verify these are the files we want:
```
"ae_network_connection_exception.gemspec",
"test/ae_network_connection_exception_test.rb",
"test/test_helper.rb",
"README.md",
"Rakefile",
"lib/ae_network_connection_exception.rb",
"lib/ae_network_connection_exception",
"lib/ae_network_connection_exception/version.rb",
"LICENSE.txt"
```